contrib/clojure-package/examples/infer/predictor/src/infer/predictor_example.clj

```diff
@@ -99,3 +99,9 @@
       (:help options) (println summary)
       (some? errors) (println (join "\n" errors))
       :else (run-predictor options))))
+
+(comment
+  (run-predictor {:model-path-prefix "models/resnet-18/resnet-18"
```

Comment:
This seems to be unrelated to the BERT work. Should it be done in another PR?

Reply:
It is related only because it came up while exploring ways to predict with models that aren't saved from disk. I left it in because otherwise one doesn't have a way to run this code from a REPL. I can remove it from this PR but I don't expect I have time right now to shepherd another PR. Would you like me to do that?

Reply:
It should be fine I think